osu.Game/Screens/Play/HUD/HoldForMenuButton.cs

```diff
@@ -99,9 +111,11 @@ protected override void Update()
                 Alpha = 1;
             else
             {
+                float minAlpha = AutoHide.Value ? .001f : .07f;
```

Comment:
I tried allowing the alpha to clamp to 0, (and enabling `AlwaysPresent` on the container), however ran into issues with testing as the interpolation below seemed to never reach a fixed value (and `IsPresent` is useless when `AlwaysPresent` is enabled`)